### PR TITLE
fix: load staticwebapp.config.json schema locally when timeout

### PR DESCRIPTION
**Pull request by cjk7989**
Added a logic to load static-web-apps-cli/schema/staticwebapp.config.json as a backup to solve issue #754 and #802

### CODE REVIEW

File: schema/staticwebapp.config.json [hunk 1]
```diff
@@ -0,0 +1,613 @@
...
+      "description": "Platform configuration",
+      "properties": {
+        "apiRuntime": {
+          "type": "string",
+          "enum": [
+            "dotnet:3.1",
+            "dotnet:6.0",
+            "dotnet-isolated:6.0",
+            "dotnet-isolated:7.0",
+            "node:12",
```
Comment (pocki) [inline schema/staticwebapp.config.json:592]: Latest platforms seems to be missing compared to Azure Functions: 
`az functionapp list-runtimes --os linux --query "[].{stack:join(':', [runtime, version]), LinuxFxVersion:linux_fx_version, SupportedFunctionsVersions:to_string(supported_functions_versions[])}" --output table`

Comment (cjk7989) [reply schema/staticwebapp.config.json:592]: Will investigate the details, thanks!

**Comment by anfibiacreativa** [comment]
Hi @cjk7989 although I agree that this PR at least decouples concerns: 1 thing is not to find the schema, another thing is for the configuration not to conform to the schema. 

Can we please revert the previous changes https://github.com/Azure/static-web-apps-cli/commit/f07e351e5c693d482061829119bcd1f37b92abda on the grounds of: do we really want to block customers from running their applications when there is a misconfiguration? What is the rationale to go from warning to error?

File: schema/staticwebapp.config.json (hunk 1 above)
Comment (cjk7989) [reply schema/staticwebapp.config.json:592]: Have updated supported platform "dotnet-isolated:8.0" in PR https://github.com/Azure/static-web-apps-cli/pull/809.

**Pull request merged**